x-pack/plugins/triggers_actions_ui/public/application/sections/field_browser/components/field_items/field_items.tsx

```diff
@@ -49,15 +50,12 @@ export const getFieldItemsData = ({
       if (categoryBrowserFields.length > 0) {
         fieldItemsAcc.push(
           ...categoryBrowserFields.map(({ name = '', ...field }) => {
-            if (!showDescriptionColumn && !!field.description) {
-              showDescriptionColumn = true;
-            }
             return {
               name,
               type: field.type,
-              description: field.description ?? '',
+              description: (EcsFlat as Record<string, EcsMetadata>)[name]?.description ?? '',
```

Comment:
Just for the fun of it, can you write a special function to extract the description and then we can add some unit test around it just to be safe.

Reply:
Fixed here: ee241cdd6ddd3b4561904114b33fee517c81483b